src/sentry/integrations/pagerduty/actions/notification.py

```diff
@@ -91,7 +91,7 @@ def get_services(self) -> Sequence[PagerDutyService]:
             service
             for service in PagerDutyService.objects.filter(
                 organization_id=self.project.organization_id, integration_id__in=integration_ids
-            )
+            ).values_list("id", "service_name")
```

Comment:
The `get_services()` method is also used on line 3 which is where the JSON encode error is coming from as the models are put into a `choices` list.